cli.js

```diff
@@ -221,11 +202,4 @@ function init(args, options) {
 sudoBlock();
 updateNotifier({pkg: pkg}).notify();
 
-if (process.stdin.isTTY) {
-	init(args, options);
-} else {
-	getStdin(function (data) {
-		[].push.apply(args, data.trim().split(/\r?\n/));
-		init(args, options);
-	});
-}
+init(subarg(cli.input, options)._, cli.flags);
```

Comment:
Do I have to pass in the `options` object (declared at the beginning of the file) in here?